config/application.rb

```diff
@@ -78,6 +78,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join("lib", "miq_automation_engine", "models", "mixins")
     config.autoload_paths << Rails.root.join("app", "controllers", "mixins")
     config.autoload_paths << Rails.root.join("lib")
+    config.autoload_paths << Rails.root.join("lib", "services")
```

Comment:
are we going to have that many extensions for sharing services that it warrants a whole new lib directory?